Plugins/UXTools/UXTools.uplugin

```diff
@@ -1,15 +1,15 @@
 {
 	"FileVersion": 3,
-	"Version": 1,
-	"VersionName": "1.0",
+	"Version": 2,
+	"VersionName": "0.9.0",
```

Comment:
Out of curiosity, did upgrading to UX Tools 0.10 have too many breaking changes?

Reply:
It was 0.9 early prerelease, but marked as 1.0 (that version was never released). I updated to the latest 0.9 to resolve all issues.